enforcer/params_utils.py

```
@@ -0,0 +1,35 @@
+import logging
+import os
+import re
+from typing import Dict, Optional
+
+from pydantic.types import SecretStr
+
+def get_env_replacement(value: str) -> Optional[str]:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)